Scripts/Linux/BuildScalarForLinux.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I tested `BuildScalarForLinux.sh` and `RunUnitTests.sh` on my Linux machine, and both worked.

Running the compiled output didn't go so hot. Perhaps we can load the Platform, since you do so much to populate it here in this PR?

```
/_git/scalar/src/Scripts/Linux$ ../../../out/Scalar/bin/Debug/netcoreapp3.1/linux-x64/scalar
Unhandled exception. System.NotImplementedException: The method or operation is not implemented.
   at Scalar.PlatformLoader.ScalarPlatformLoader.Initialize() in /_git/scalar/src/Scalar.Common/Platforms/PlatformLoader.cs:line 25
   at Scalar.Program.Main(String[] args) in /_git/scalar/src/Scalar/Program.cs:line 15
Aborted (core dumped)
```